tests/test_openhands_conversation_export_scripts.py

```
@@ -0,0 +1,136 @@
+import tempfile
+import unittest
+from pathlib import Path
+
+
+REPO_ROOT = Path(__file__).resolve().parents[1]
+SCRIPTS_DIR = REPO_ROOT / "skills" / "openhands-conversation-export" / "scripts"
+
+
+class TestOpenHandsConversationExportScripts(unittest.TestCase):
+    def test_truncate_json_redacts_and_truncates(self) -> None:
+        token = "ghu_abcdefghijklmnopqrstuvwxyz123456"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "token"

(S105)